data-prepper-plugins/otel-metrics-raw-processor/README.md

```diff
@@ -9,6 +9,98 @@ processor:
     - otel_metrics_raw_processor
 ```
 
+## Configuration
+It is possible to create explicit representations of histogram buckets and their boundaries. This feature can be controlled with the following parameters:
+
+```yaml
+  processor:
+    - otel_metrics_raw_processor:
+        calculate_histogram_buckets: true
```

Comment:
I keep wondering about the default here. One of the goals of Data Prepper is to perform as much processing as possible during streaming so that OpenSearch does not need to do it. So I'd think that Data Prepper would typically calculate both bucket types. And if that is the case, then I think the default should be true.

What do you think?

Reply:
I was wandering about the same thing. And because the calculation of the bucket boundaries is not something that a user would typically expect because it adds something on top of what OpenTelemetry provides I initially thought it might be better to turn it off. But I can see the point of offloading the processing and this line of reasoning makes more sense in this context. I will turn it on by default.